setup/mysql.sql

```diff
@@ -145,10 +147,25 @@ INSERT INTO `vexim`.`users`
     smtp, pop, realname, type, admin
 )
 VALUES 
-(   '1', 'siteadmin', 'siteadmin', '$1$12345678$2lQK5REWxaFyGz.p/dos3/', '65535', '65535', '', '',  'SiteAdmin', 'site', '1'
+(   '1', 'siteadmin', 'siteadmin', '$6$uR.EiB1dj5rrvwMF$Qh5LgdjOZavKXwhi9IF0Yuzu7qxsG.dLTTB8e./55ZRNfBuZVLnfUSOEXa0oWT6174myO.WYkOy83HYWAKPbK/', '65535', '65535', '', '',  'SiteAdmin', 'site', '1'
```

Comment:
As we change the password after the first login, the crypt scheme technically doesn't matter. What I thought of, if someone checks out the basics of password hashing and sees that $1$ refers to md5, he might think that if we decide to use this for the super-admin password this will be sufficient for his user's passwords as well.

For compatibility we shouldn't worry too much, it requires php 5.3.2 (http://www.w3schools.com/php/func_string_crypt.asp), which is already EOL (https://secure.php.net/eol.php).

https://www.bytepark.de/blog/2013/10/list-of-current-php-version-in-major-linux-distros/

For the crypt() function, glibc 2.7 or higher is needed, which was widely introduced some time back:
    Debian 5.0 lenny
    Fedora 8 werewolf
    Gentoo 2008.0
    Mandriva 2008.1
    Slackware 12.1 ... 12.2
    Turbolinux 12D magny-cours
    ubuntu 8.04 LTS Hardy Heron